### DISCUSSION TITLE
Can't understand if fields can be arrays

### DISCUSSION THREAD

**Discussion by g1sbi** [pocketbase/pocketbase#2200] [Q&A]
I have a URL field that I'd like to be an array instead of a single value, since every entry can have multiple URLs. However, I can't seem to understand how or if I can even do that. What's the correct way to implement this?

**Comment by ganigeorgiev** [comment]
The `url` type field supports only single value, aka. it can't be an array.

If you want to store array value, you can use the `json` type field (_note that it doesn't have any url validations_) OR store the url as part of another collection and point from/to it with a `relation` field.
